integrated_channels/moodle/transmitters/content_metadata.py

```diff
@@ -36,8 +38,4 @@ def _prepare_items_for_transmission(self, channel_metadata_items):
         return items
 
     def _serialize_items(self, channel_metadata_items):
-        """
```

Comment:
What's the reason for removing this doc string?

Reply:
lol oops.